python/README.md

```diff
@@ -71,17 +72,54 @@ for the latest API documentation.
     
 ### Usage Instructions
 
-To use the Python library:
+#### Install From PyPI
 
-1. Install Python 3.4 (or later) and pip.
-2. Install the Python requirements:
+1. Install Python 3.7 (or later) and pip.
+2. Install the `fusione-engine-client` module _including_ extra requirements used by the analysis and processing tools:
```

Comment:
While more technically correct, I think this language is confusing. We should just say "it installs the analysis and processing tools and the associated dependencies"

Reply:
Ok, changed to simply:
```
2. Install the `fusione-engine-client` module, including all analysis and data processing tools:
```